lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller.pm

```diff
@@ -124,7 +37,7 @@ sub _check_error {
     return (400, "Subfolder has invalid characters") if $subfolder && $subfolder =~ m!/!;
     return (400, "Filename has invalid characters")  if $filename  && $filename =~ m!/!;
 
-    return (404, "Invalid Project") if $project && !-d Mojo::File->new($home, $project);
+    return (404, "Invalid Project {" . $project . "}") if $project && !-d Mojo::File->new($home, $project);
     return 0;
 }
```

Comment:
This whole base class serves no real purpose anymore and should be replaced with Mojolicious best practices.